rotkehlchen/constants/assets.py

```diff
@@ -27,6 +28,7 @@
 A_YFI = Asset('eip155:1/erc20:0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e')
 A_PETH = Asset('eip155:1/erc20:0xf53AD2c6851052A81B42133467480961B2321C09')
 A_USDT = Asset('eip155:1/erc20:0xdAC17F958D2ee523a2206206994597C13D831ec7')
+A_OPTIMISM_USDT = Asset('eip155:10/erc20:0x94b008aA00579c1307B0EF2c499aD98a8ce58e58')
```

Comment:
Is this only used in tests and not production decoders? Then move it to `tests/utils/constants.py`